common/model/opening-hours.js

```diff
@@ -38,29 +38,30 @@ export const galleryOpeningHours: OpeningHours = {
     {dayOfWeek: 'Saturday',  opens: '10:00', closes: '18:00'},
     {dayOfWeek: 'Sunday',    opens: '11:00', closes: '18:00'}
   ],
-  exceptionalOpeningHours: [
-    {overrideDate: new Date('April 2, 2018'), dayOfWeek: 'Monday', opens: '10:00', closes: '17:00'},
-    {overrideDate: new Date('May 7, 2018'), dayOfWeek: 'Monday', opens: '10:00', closes: '17:00'},
-    {overrideDate: new Date('May 28, 2018'), dayOfWeek: 'Monday', opens: '10:00', closes: '17:00'},
-    {overrideDate: new Date('December 25, 2018'),  dayOfWeek: 'Tuesday', closes: '00:00', note: 'Galleries closed'},
-    {overrideDate: new Date('December 26, 2018'),  dayOfWeek: 'Wednesday', closes: '00:00', note: 'Galleries closed'}
+  exceptional: [
+    {overrideDate: '2018-04-02', dayOfWeek: 'Monday', opens: '10:00', closes: '18:00'}
```

Comment:
Is `dayOfWeek` necessary as it could(/maybe should) be inferred from the `overrideDate`?